src/components/MobileNav.js

```diff
@@ -27,7 +31,33 @@ import { ReactComponent as CloseIcon } from 'images/close.svg';
 import { ReactComponent as ActiveHome } from 'images/active-home.svg';
 import { ReactComponent as InactiveHome } from 'images/inactive-home.svg';
 
-const CDPDropdown = ({ children }) => {
+const NavbarIcon = ({ owned, label, ratio, connected }) => (
+  <Flex
+    flexDirection="column"
+    alignItems="center"
+    justifyContent="center"
+    bg={label ? 'teal.500' : 'black.500'}
+    borderRadius="default"
+    height="50px"
+    width="64px"
+  >
+    {label ? (
+      <Fragment>
+        <Text t="p6" fontWeight="bold" color={owned ? 'white' : 'darkPurple'}>
+          {label}
+        </Text>
+        <RatioDisplay fontSize="1.3rem" ratio={ratio} active />
+      </Fragment>
+    ) : connected ? (
+      <ActiveHome />
+    ) : (
+      <InactiveHome />
+    )}
+  </Flex>
+);
+
+const CDPDropdown = memo(function({ iconData, children }) {
```

Comment:
@levity I used `memo` because I thought it was required with `useEffect`. Since that is now removed, do I still need `memo` here?

Reply:
`memo` is still helpful to use in general if you want a component not to change if its props don’t change.